docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -0,0 +1,461 @@
+				{
+					label: 'advanced',
+					items: [
+						{ label: 'Project Structure', link: '/guides/project-structure/' },
+					],
+				},
+			],
+		},
+	]}
+/>
+
+### Customizing autogenerated links in frontmatter
+
+Use the [`sidebar` frontmatter field](/reference/frontmatter/#sidebar) to customize autogenerated links.
+
+This field allows you to set a [custom label](/reference/frontmatter/#label) or add a [badge](/reference/frontmatter/#badge) for a page link, to [hide](/reference/frontmatter/#hidden) a link from the sidebar or to define a [custom order](/reference/frontmatter/#order) for all page links.
+
+```md
+---
+title: My page
+sidebar:
+  # Set a custom label for the link
+  label: Custom sidebar label
+  # Show the link in the sidebar
+  hidden: false
+  # Set a custom order for the link (lower numbers are displayed higher up)
+  order: 2
+  # Add a badge to the link
+  badge:
+    text: Deprecated
+    variant: caution
+---
+```
+
+A sidebar including an autogenerated group for the `guides` directory containing a page with the above frontmatter will generate the following sidebar:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guides',
+			items: [
+				{ label: 'A page', link: '#' },
+				{
+					label: 'Custom sidebar label',
+					link: '#',
+					badge: { text: 'Deprecated', variant: 'caution' },
+				},
+				{ label: 'Another page', link: '#' },
+			],
+		},
+	]}
+/>
+
+:::note
+The `sidebar` frontmatter configuration is only used for autogenerated links and will be ignored for manually defined links.
+:::
+
+## Internationalization
+
+Manually defined link and group labels can be translated using the `translations` property to define a set of labels for each supported language while the `label` property will act as the default locale.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Guides',
+			translations: { es: 'Guías' },
+			items: [
+				{
+					label: 'Components',
+					translations: { es: 'Componentes' },
+					link: '/guides/components/',
+				},
+				{
+					label: 'Internationalization (i18n)',
+					translations: { es: 'Internacionalización (i18n)' },
+					link: '/guides/i18n/',
+				},
+			],
+		},
+	],
+});
+```
+
+Browsing the documentation in Spanish will generate the following sidebar:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Guías',
+			items: [
+				{ label: 'Componentes', link: '/guides/components/' },
+				{ label: 'Internacionalización (i18n)', link: '/guides/i18n/' },
+			],
+		},
+	]}
+/>
+
+## Collapsing groups
+
+Groups of links can be collapsed by default by setting the `collapsed` property to `true`.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Guides',
+			// Collapse the group by default.
+			collapsed: true,
+			items: [
+				{ label: 'Components', link: '/guides/components/' },
+				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
+			],
+		},
+	],
+});
+```
+
+The above configuration will generate the following sidebar:
```

Comment:
Heh. This actually already got the wording more or less exactly that I suggested above! I guess we should flip the `above` so it’s fully consistent:

```suggestion
The configuration above generates the following sidebar:
```